webpack/components/OpenscapRemediationWizard/steps/ReviewHosts.js

```diff
@@ -161,8 +162,8 @@ const ReviewHosts = () => {
   const columns = {
     name: {
       title: __('Name'),
-      wrapper: ({ id, display_name: displayName }) => (
-        <a href={foremanUrl(`hosts/${id}`)}>{displayName}</a>
+      wrapper: ({ name, display_name: displayName }) => (
+        <a href={foremanUrl(`${getHostsPageUrl(true)}/${name}`)}>{displayName}</a>
```

Comment:
I suggested the same mistake I did in yesterdays PR :upside_down_face: 
I meant `HOSTS_PATH` from `import { HOSTS_PATH } from 'foremanReact/routes/Hosts/constants';`
HOSTS_PATH is for `/new/hosts` , for single hosts 
getHostsPageUrl is for the experimental index host page

Reply:
But why is that a mistake? The link is built right and we do use it in other places as well: 

https://github.com/theforeman/foreman_openscap/blob/master/webpack/components/OpenscapRemediationWizard/ViewSelectedHostsLink.js#L17

https://github.com/theforeman/foreman_openscap/blob/master/webpack/components/OpenscapRemediationWizard/steps/ReviewRemediation.js#L123

Reply:
I dont think `getHostsPageUrl` is supposed to be used this way anyway 
```
export const getHostsPageUrl = displayNewHostsPage =>
  displayNewHostsPage ? '/new/hosts' : '/hosts';

export const useForemanHostsPageUrl = () => {
  const { displayNewHostsPage } = useForemanSettings();
  return getHostsPageUrl(displayNewHostsPage);
};
```
`useForemanHostsPageUrl` is using the setting `displayNewHostsPage` which is for the index page, and not for the show page

Reply:
Created https://github.com/theforeman/foreman/pull/10174 so foreman context will use the settings for both index and show page for host url